backend/tracim_backend/lib/core/live_messages.py

```diff
@@ -0,0 +1,54 @@
+import json
+import typing
+
+from gripcontrol import GripPubControl
+
+# TODO - G.M - 2020-05-14 - Use default event "message" for TLM to be usable with
+# "onmessage" EventSource Object in javascript.
+from tracim_backend import CFG
+from tracim_backend.models.event import Message
+from tracim_backend.views.core_api.schemas import LiveMessageSchema
+
+TLM_EVENT_NAME = "message"
+
+
+class ServerSideEvent(object):
+    """Create a ServerSideEvent String Like"""
+
+    def __init__(
+        self, data: typing.List[str], event: typing.Optional[str] = None,
+    ):
+        self.data = data
+        self.event = event
+        self._buffer = ""
+        self._convert_to_sse_string()
+
+    def _convert_to_sse_string(self):
+        if self.event:
+            self._buffer += "event: {}\n".format(self.event)
+        for row in self.data:
+            self._buffer += "data: {}\n".format(row)
+
+        self._buffer += "\n"
+
+    def __str__(self):
+        return self._buffer
```

Comment:
This should be simpler with a simple function, isn't it? It could also do the 'json.dumps()' call with the right arguments to have only one `data:` line.

Reply:
Idea here, is just to have an simple unopiniated implementation of sse protocol.

I don't really like the idea of having a method that does thing at multiple protocol layer at once but that can be discussed as the protocol is really simple here.

Reply:
i adjust the code a bit.

Reply:
Uh oh, there is a bug here that I didn't see!
why is data a list? What if there is a string with new lines in this list? Why not just have a data string?

Reply:
but I agree that this code should not do anything related to json here

Reply:
There is a data list to allow multiline sse, because it's an unopiniated implementation, but i can remove this, if needed.

Reply:
IMHO you should accept a regular string and split it by new lines